controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/VersionStatusUpdater.java

```diff
@@ -47,14 +48,15 @@ protected double maintain() {
     }
 
     static SystemMonitor.Confidence convert(VespaVersion.Confidence confidence) {
-        switch (confidence) {
-            case aborted: return aborted;
-            case broken:  return broken;
-            case low:     return low;
-            case normal:  return normal;
-            case high:    return high;
-            default: throw new IllegalArgumentException("Unexpected confidence '" + confidence + "'");
-        }
+        return switch (confidence) {
+            case aborted -> aborted;
+            case broken -> broken;
+            case low -> low;
+            case legacy -> legacy;
+            case normal -> normal;
+            case high -> high;
+            default -> throw new IllegalArgumentException("Unexpected confidence '" + confidence + "'");
```

Comment:
I think `default` can be removed since the switch is exhaustive.

Reply:
Fixed, thanks.